neptune/new/internal/utils/deprecation.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from functools import wraps
+
+from neptune.new.exceptions import NeptuneInitParametersCollision
+from neptune.new.internal.utils.logger import logger
+
+
+def deprecated_id_parameter(*, deprecated_kwarg_name):
```

Comment:
How about having generic method and passing argument name?

Reply:
Ok